src/betterproto/plugin/models.py

```diff
@@ -679,6 +688,15 @@ def __post_init__(self) -> None:
         if self.client_streaming or self.server_streaming:
             self.output_file.typing_imports.add("AsyncIterator")
 
+        # add imports required for request arguments timeout, deadline and metadata
+        self.output_file.typing_imports.add("Optional")
+        self.output_file.imports_type_checking_only.add(
+            "from betterproto.grpc.grpclib_client import _MetadataLike"
+        )
+        self.output_file.imports_type_checking_only.add(
+            "from grpclib.metadata import Deadline as Deadline"
```

Comment:
```suggestion
            "from grpclib.metadata import Deadline"
```
The as here shouldn't be neccesary, the type shouldn't be re-exported by the protobuf file